Arquivos/helpers/memoria.py

```diff
@@ -66,17 +68,27 @@ def deallocate(self, index, file):
             _file = self.data[i]
             if _file.name == file:
                 find = True
-                self._clean_memory(_file.indexes)
+                # self._clean_memory(_file.indexes)
+                self.delete_in_cascade(_file.indexes)
```

Comment:
Eu tava usando o padrão de quando uma função for de uso interno inserir na frente um `_`, nesse caso ficaria `_delete_in_cascade(_file.indexes)`.